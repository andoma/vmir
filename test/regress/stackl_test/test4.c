
int main()
{
    int ii;
    for (ii = 0 ; ii<5 ; ii = ii+1)
    {
        printf("for loop\n");
    }
    return 0;
}
