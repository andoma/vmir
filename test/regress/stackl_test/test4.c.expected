for loop
for loop
for loop
for loop
for loop
